public/js/components/Frames.js

```diff
@@ -66,6 +66,9 @@ const Frames = React.createClass({
 
   renderFrames() {
     let { frames } = this.props;
+    if (!frames) {
+      return null;
+    }
```

Comment:
I like it, i'm happy to merge in the check. It was a small nit, but you persuaded me it is good to be safe